### PR TITLE
add more systemctl commands

### PR DESCRIPTION
**Pull request by pankajbhanu**
hi, i added some more commonly used systemd commands from https://gist.github.com/DStorck/cd7d0bd257dc672df44b015af4db31ab

### CODE REVIEW

File: systemctl [hunk 1]
```diff
@@ -16,16 +16,53 @@ systemctl stop foo.service
...
+# To reload systemd manager configuration:
+systemctl daemon-reload
+
 # To enable a service to startup on boot:
 systemctl enable foo.service
 
 # To disable a service to startup on boot:
 systemctl disable foo.service
 
+#To see all active units, add --all for everything
```
Comment (waldyrious) [inline systemctl]: This line needs a space between the `#` and `To`, and is missing the `:` at the end as well. Same for the lines below.

File: systemctl [hunk 2]
```diff
@@ -1,31 +1,65 @@
...
+# To reload systemd manager configuration:
+systemctl daemon-reload
+
 # To enable a service to startup on boot:
 systemctl enable foo.service
 
 # To disable a service to startup on boot:
 systemctl disable foo.service
 
+# To start/restart/stop per-user service
```
Comment (waldyrious) [inline systemctl]: Still missing a colon at the end here :)

**Comment by pankajbhanu** [comment]
@waldyrious any plans  on merging this pull request

**Comment by chrisallenlane** [comment]
Hi, guys

Sorry - there's a lot here, and I haven't had time to review it yet. (While I don't _thoroughly_ test each cheat submitted, I at least sanity-check them, and I'll have to do some Googling before I can understand the tips submitted here).

My delay in merging this isn't a reflection on the quality of the contribution - I've just been really busy with the day job.

I should be able to get to this sometime in the next few days. Sorry for the delay!

**Comment by waldyrious** [comment]
And just to make things clear, I don't have the ability to merge PRs on this repo. I was reviewing just as a fellow contributor interested in this project :)

**Comment by chrisallenlane** [comment]
Hi, guys. Sorry for the lengthy delay on this. This has now been merged. Thanks!

**Pull request merged**